app/controllers/api/v1/rooms_configurations_controller.rb

```diff
@@ -3,6 +3,7 @@
 module Api
   module V1
     class RoomsConfigurationsController < ApiController
+      # TODO: samuel - Review this, RoomsConfig#index is needed when user with ManageRooms what to access another users room
       before_action only: %i[index] do
         ensure_authorized(%w[CreateRoom ManageSiteSettings ManageRoles], friendly_id: params[:friendly_id])
       end
```

Comment:
Is this really needed?

The data from /api/v1/rooms_configurations:
`{"data":{"record":"false","muteOnStart":"optional","guestPolicy":"optional","glAnyoneCanStart":"optional","glAnyoneJoinAsModerator":"optional","glRequireAuthentication":"optional","glModeratorAccessCode":"optional","glViewerAccessCode":"optional"}}`

Reply:
Quick thought before the weekend: maybe adding an OR in ensure_authorized would be an idea?
This user either has CreateRoom || ManageSiteSettings || etc.